src/components/connect-request.js

```diff
@@ -50,39 +67,77 @@ const connectRequest = (mapPropsToConfig, options = {}) => (WrappedComponent) =>
             return this._wrappedInstance;
         }
 
-        cancelPendingRequests() {
+        // returns [[ "keys", "to", "cancel"], [{ config objects to request }]]
+        _diffConfigs(prevConfigs, configs) {
```

Comment:
Can you move this method out of the component class to the top of the file to reinforce that it's a pure function that doesn't read from any of the component instance props/state/variables? Then I think it makes sense to drop the prefixed `_`.